src/bundle/Resources/public/js/CKEditor/custom-tags/inline-custom-tag/inline-custom-tag-editing.js

```diff
@@ -0,0 +1,146 @@
+            isInline: true,
+            allowIn: 'inlineCustomTag',
+            allowChildren: '$text',
+        });
+    }
+
+    defineConverters() {
+        const { conversion } = this.editor;
+
+        conversion.for('editingDowncast').elementToElement({
+            model: 'inlineCustomTag',
+            view: (modelElement, { writer: downcastWriter }) => {
+                const container = downcastWriter.createContainerElement('span', {
+                    'data-ezelement': 'eztemplateinline',
+                    'data-ezname': modelElement.getAttribute('customTagName'),
+                    class: 'ez-custom-tag',
+                });
+
+                return toWidget(container, downcastWriter);
+            },
+        });
+
+        conversion.for('dataDowncast').elementToElement({
+            model: 'inlineCustomTag',
+            view: (modelElement, { writer: downcastWriter }) => {
+                const container = downcastWriter.createContainerElement('span', {
+                    'data-ezelement': 'eztemplateinline',
+                    'data-ezname': modelElement.getAttribute('customTagName'),
+                });
+                const config = downcastWriter.createUIElement('span', { 'data-ezelement': 'ezconfig' }, function(domDocument) {
+                    const domElement = this.toDomElement(domDocument);
+
+                    domElement.innerHTML = Object.entries(modelElement.getAttribute('values')).reduce((total, [attribute, value]) => {
+                        const attributeValue = value !== null ? value : '';
+                        const ezvalue = `<span data-ezelement="ezvalue" data-ezvalue-key="${attribute}">${attributeValue}</span>`;
+
+                        return `${total}${ezvalue}`;
+                    }, '');
+
+                    return domElement;
+                });
+
+                downcastWriter.remove(downcastWriter.createRangeIn(container));
+                downcastWriter.insert(downcastWriter.createPositionAt(container, 0), config);
+
+                return container;
+            },
+        });
+
+        conversion.for('upcast').elementToElement({
+            view: {
+                name: 'span',
+                attributes: {
+                    'data-ezelement': 'eztemplateinline',
+                },
+            },
+            model: (viewElement, { writer: upcastWriter }) => {
+                if (viewElement.getAttribute('data-eztype') === 'style') {
+                    return;
+                }
+
+                const configElement = viewElement.getChild(1);
+                const configValuesIterator = configElement.getChildren();
+                const customTagName = viewElement.getAttribute('data-ezname');
+                const values = {};
+
+                for (const configValue of configValuesIterator) {
+                    const value = (configValue.getChild(0) && configValue.getChild(0).data) || null;
```

Comment:
What about:
```suggestion
                    const value = configValue.getChild(0)?.data ?? null;
```
It works a little bit differently because e.g. it would return `0` instead of `null` in case `data` is equal to `0`. But do we want to return null in case `data` is `0`? Or maybe `data` is always a string etc? Could you explain?